### PR TITLE
[4.0] com_newsfeeds menu items

### PR DESCRIPTION
**Pull request by brianteeman**
Due to an sql error the complete newsfeeds menu items were not being displayed in the menu or the components dashboard

### before
image

### after
image

### CODE REVIEW

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 8adb4afd2ec583174eb3c21509b12a62de51b4ef<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26092">issues.joomla.org/tracker/joomla-cms/26092</a>.</sub>

File: administrator/components/com_admin/sql/updates/mysql/4.0.0-2019-08-30.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+UPDATE `#__menu` SET `parent_id`='12' WHERE `id`=13;
```
Comment (wilsonge) [inline administrator/components/com_admin/sql/updates/mysql/4.0.0-2019-08-30.sql]: The update file needs to be removed here. People upgrading from J3 are going to have different ID's here https://github.com/joomla/joomla-cms/blob/staging/installation/sql/mysql/joomla.sql#L1419-L1420 and they are currently correct. So this is just a bug in our J4 installation data. Nothing to fix for people upgrading

**Comment by brianteeman** [comment]
update sql files removed as requested - i only put them because...

**Comment by wilsonge** [comment]
95% of the time we should have them since in alpha 11 we're claiming to have a give or take working upgrade path (in that there's expected errors as the template swaps). This is the 5% where there's nothing to fix for those upgrading.

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**